src/app/config.rs

```diff
@@ -190,9 +194,8 @@ pub const ENV_ADMIN_LISTEN_ADDR: &str = "LINKERD2_PROXY_ADMIN_LISTEN_ADDR";
 pub const ENV_METRICS_RETAIN_IDLE: &str = "LINKERD2_PROXY_METRICS_RETAIN_IDLE";
 const ENV_INBOUND_CONNECT_TIMEOUT: &str = "LINKERD2_PROXY_INBOUND_CONNECT_TIMEOUT";
 const ENV_OUTBOUND_CONNECT_TIMEOUT: &str = "LINKERD2_PROXY_OUTBOUND_CONNECT_TIMEOUT";
-const ENV_INBOUND_CONNECT_BACKOFF: &str = "LINKERD2_PROXY_INBOUND_CONNECT_BACKOFF";
-const ENV_OUTBOUND_CONNECT_BACKOFF: &str = "LINKERD2_PROXY_OUTBOUND_CONNECT_BACKOFF";
-
+const ENV_INBOUND_CONNECT_BASE: &str = "INBOUND_CONNECT";
```

Comment:
Because this is not an environment variable in the sense that the other variables are in this file, I think we should call this something different -- perhaps just `INBOUND_CONNECT_BASE`.

This constant is used as part of generating the configuration as a whole, but it is not something that is dependent on the environment.

This, along with outbound connect and control base could be put at the top of the file like other constants are throughout the project ([e.g.](https://github.com/linkerd/linkerd2-proxy/blob/5f58883a7e34bc449f0b0e8c6f997018c5eda32a/src/app/mod.rs#L21-L22))

Let me know if that makes sense or if you think otherwise!